packages/core/src/utils/CollectLabels.ts

```diff
@@ -64,37 +63,32 @@ type Output = {
 
 /**
  * Call MathJax to render __non-empty__ labels.
- * NOTE: this function is memoized.
  */
-const tex2svg = memoize(
```

Comment:
Could you clarify why you removed this `memoize`?

Reply:
Yes, #535 noted this issue, but there's no PR that fixes it. This fix doesn't close the issue though, because when downloading the SVGs (which is currently broken in Edgeworth, to be addressed in another PR) the labels still disappear.